domain/entity/latest_entry.go

```diff
@@ -0,0 +1,91 @@
+package entity
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/appengine/datastore"
+)
+
+const (
+	LatestEntryCodeTamai    = "tamai-sd"
+	LatestEntryCodeMomota   = "momota-sd"
+	LatestEntryCodeAriyasu  = "ariyasu-sd"
+	LatestEntryCodeSasaki   = "sasaki-sd"
+	LatestEntryCodeTakagi   = "takagi-sd"
+	LatestEntryCodeHappyclo = "happyclo"
+)
+
+type (
+	// LatestEntry for confirm last updated entry url
+	LatestEntry struct {
+		ID        string `datastore:"-" goon:"id"`
+		Code      string
+		URL       string
+		CreatedAt time.Time
+		UpdatedAt time.Time
+	}
+
+	// LatestEntryRepository interface
+	LatestEntryRepository interface {
+		Save(*LatestEntry) error
+		GetTamaiURL() string
+		GetMomotaURL() string
+		GetAriyasuURL() string
+		GetSasakiURL() string
+		GetTakagiURL() string
+		GetHappycloURL() string
+	}
+)
+
+func (l *LatestEntry) SetCreatedAt(t time.Time) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method LatestEntry.SetCreatedAt should have comment or be unexported